github/github.go

```diff
@@ -167,6 +169,8 @@ type Client struct {
 	// User agent used when communicating with the GitHub API.
 	UserAgent string
 
+	PackageVersion string
```

Comment:
I thought that exposing this on the `Client` would help with (a) people constructing customer `User-Agent` headers and (b) me testing the default `User-Agent`!